Dockerfile

```diff
@@ -67,6 +67,8 @@ RUN cd pytorch && \
       NO_DISTRIBUTED=1 NO_TEST=1 NO_MIOPEN=1 USE_MKLDNN=0 USE_OPENCV=OFF USE_LEVELDB=OFF \
       python setup.py install && python setup.py clean
 
+# PS: No need to build LibTorch from source as it is already present in the docker
```

Comment:
Remove this comment and instead in at line 59 add something like. "This will build both the caffe2 libraries needed by the caffe2 netdef backand and the libtorch library needed by the pytorch backend.

Reply:
Will do